pkgs/build-support/buildenv/builder.pl

```diff
@@ -110,7 +110,7 @@ sub findFiles {
     }
 
     unless (-d $target && ($oldTarget eq "" || -d $oldTarget)) {
-        if ($ignoreCollisions) {
+        if ($ignoreCollisions && !checkCollision($oldTarget, $target)) {
```

Comment:
my reading of the code is that we have three possibilities:

1. ignore collisions with a warning
2. silently allow collisions, only if file contents are identical
3. don't allow any collisions at all

`ignoreCollisions` tells us if we're in case 1 or not.  Then only if not, `checkCollisionContents` tells us if we're in 2 or 3.

Since all I'm aiming for is silencing meaningless warnings in case 1, I didn't think `checkCollisionContents` was relevant.